src/core/chains.ts

```
@@ -1,281 +1,33 @@
-  2522: 'https://rpc.testnet.frax.com',
-  919: 'https://sepolia.mode.network',
-  59902: 'https://sepolia.metis.io/?owner=59902',
-  2358: 'https://api.sepolia.kroma.network',
-  999999999: 'https://sepolia.rpc.zora.energy',
-  44787: 'https://alfajores-forno.celo-testnet.org',
-  5: 'https://rpc.ankr.com/eth_goerli',
-  17000: 'https://ethereum-holesky.publicnode.com',
-  545: 'https://testnet.evm.nodes.onflow.org',
+  1329: 'https://evm-rpc.sei-apis.com',
+  1328: 'https://evm-rpc-testnet.sei-apis.com',
+  713_715: 'https://evm-rpc-arctic-1.sei-apis.com'
```

Biome (1.9.4)

[error] Number literal with underscore is not allowed here.

Safe fix: Replace 713_715 with 713715


(lint/complexity/useSimpleNumberKeys)